lang/el.po

```diff
@@ -0,0 +1,100 @@
+msgid ""
+msgstr ""
+"Language: en\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n==1 ? 0 : 1);\n"
+
+# A place for you to credit yourself!
+# Put something like "Latin translation by Jvlivs Gaivs\n" here.
+msgid "TRANSLATION_AUTHOR\n"
+msgstr "Tassos Natsakis\n"
+
+msgid "FREE_PACKAGES_COUNT\n"
+msgid_plural "FREE_PACKAGES_COUNT\n"
+msgstr[0] "%d ελεύθερο πακετό\n"
+msgstr[1] "%d ελεύθερα πακέτα\n"
+
+msgid "NONFREE_PACKAGES_COUNT\n"
+msgid_plural "NONFREE_PACKAGES_COUNT\n"
+msgstr[0] "%d μη ελεύθερο πακέτκο\n"
+msgstr[1] "%d μη ελεύθερα πακέτα\n"
+
+msgid "RMS_HAPPY\n"
+msgstr "Μόνο ελεύθερα πακέτα - ο rms θα ήταν περήφανος!\n"
+
+msgid "RMS_DISAPPOINTED\n"
+msgstr "Πάνω από 10%% μη ελεύθερα πακέτα. Δεν εκτιμάτε την ελευθερία;\n"
+
+msgid "HELP_USAGE\n"
+msgstr "Χρήση: vrms-rpm [επιλογές]\n"
+
+msgid "HELP_OPTION_ASCII\n"
+msgstr "    Εμφανίζεται rms ASCII-art οταν δεν βρίσκονται μη-ελεύθερα πακέτα,\n"
+       "    ή όταν τα μη ελεύθερα πακέτα είναι λιγότερα από 10%% από το σύνολο.\n"
+
+msgid "HELP_OPTION_COLOUR\n"
+msgstr "    Ελέγχει εάν πρέπει να χρησιμοποιηθούν ακολουθίες διαφυγής τερματικού\n"
+       "    για τον χρωματισμό της εξόδου. Η προεπιλογή είναι 'auto', η οποία παράγει\n"
+       "    χρωματισμένη έξοδο όταν γράφεται σε τερματικό, αλλά όχι όταν γράφεται σε\n"
+       "    αρχείο ή σε διασωλήνωση (pipe).\n"
+
+msgid "HELP_OPTION_DESCRIBE\n"
+msgstr "    Όταν εκτυπώνονται τα αρχεία, συμπεριελάμβανε τις περιλήψεις τους (σύντομες περιγραφές).\n"
+
+msgid "HELP_OPTION_EXPLAIN\n"
+msgstr "    Όταν εκτυπώνονται τα αρχεία, εκτύπωσε τις αδειές τους\n"
+       "    για να αιτιολογηθεί η ταξινόμηση σε ελεύθερα / μη ελεύθερα.\n"
+
+msgid "HELP_OPTION_HELP\n"
+msgstr "    Εκτύπωσε αυτή τη βοήθεια και τερμάτησε.\n"
+
+msgid "HELP_OPTION_IMAGE\n"
+msgstr "    Όπως το --ascii, αλλά εκτυπώνει μία εικόνα χρησιμοποιώντας χαρακτήρες Unicode\n"
+       "    και χαρακτήρες διαφυγής τερματικού 256-χρωμάτων.\n"
+
+msgid "HELP_OPTION_LICENCELIST\n"
+msgstr "    Προσδιορίζει την λίστα με τις καλές άδειες ή οποία θα χρησιμοποιηθεί. Το FILE μπορεί να είνα και διαδρομή\n"
```

Comment:
The `HELP_...` messages shouldn't be longer than 80 characters per line (as in, 80 characters between opening quote (`"`) and the closing newline+quote (`\n"`). You can break the message into multiple lines or move words and sentences between lines as you wish, as long as the meaning is kept.

Reply:
I think I fixed this too.